config.js

```diff
@@ -30,6 +30,7 @@ const config = {
   PINNED_LABEL_1: process.env.PINNED_LABEL_1,
   PINNED_URI_2: process.env.PINNED_URI_2,
   PINNED_LABEL_2: process.env.PINNED_LABEL_2,
+  FALLBACK_THUMBNAIL_FALLBACK: process.env.FALLBACK_THUMBNAIL_FALLBACK,
```

Comment:
Lets just name this `THUMBNAIL_FALLBACK`

Reply:
Thank you for the comment @seanyesmunt , updated.